src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -1090,4 +1097,73 @@ public function setEchoFormat($format)
     {
         $this->echoFormat = $format;
     }
+
+    /**
+     * Compile the switch statements into valid PHP.
+     * @param  string  $variable
+     *
+     * @return string
+     */
+    public function compileSwitch($variable)
+    {
+        return "<?php switch({$variable}){ ";
```

Comment:
Surely not closing the php tags here is just asking for trouble? People aren't meant to put any content between `@switch` and `@case` statements, but it might still happen by accident. If you don't close the PHP tags, this will cause a fatal error, rather than just echoing some content where you didn't mean to.